build.rs

```diff
@@ -27,180 +21,46 @@ fn main() {
-            Localizer {
-                include_regex: Regex::new(r"#include <([\w/]+\.h)>").unwrap(),
-                local_include_root,
-            }
-        }
-
-        fn localize_dir(&self, original_include: &Utf8Path, local_include: &Utf8Path) {
-            fs::create_dir(local_include)
-                .or_else(|e| match e.kind() {
-                    io::ErrorKind::AlreadyExists => Ok(()),
-                    _ => Err(e),
-                })
-                .unwrap();
-            for entry in original_include.read_dir_utf8().unwrap() {
-                let entry = entry.unwrap();
-                let local_entry = local_include.join(entry.file_name());
-                let original_entry = entry.path();
-                let ty = entry.file_type().unwrap();
-                if ty.is_dir() {
-                    self.localize_dir(original_entry, &local_entry)
-                } else if ty.is_file() {
-                    self.localize_file(original_entry, &local_entry)
-                } else {
-                    panic!("{original_entry:?} is bad file type: {ty:?}")
-                }
-            }
-        }
-
-        // TODO: trace span to better find sources of errors
-        fn localize_file(&self, original_include: &Utf8Path, local_include: &Utf8Path) {
-            let relative_local_root = local_include
-                .strip_prefix(self.local_include_root)
-                .unwrap()
-                .ancestors()
-                .skip(2)
-                .map(|_| "..")
-                .collect::<Utf8PathBuf>();
-            let original_include = fs::read_to_string(original_include).unwrap();
-            let new_include = self.include_regex.replace_all(
-                &original_include,
-                |captures: &regex::Captures<'_>| {
-                    // Do not replace if it's one of these include paths
-                    if let "stddef.h" | "stdint.h" | "stdarg.h" = &captures[1] {
-                        return captures[0].to_owned();
-                    }
-                    let path = relative_local_root.join(&captures[1]);
-                    format!("#include \"{path}\"")
-                },
-            );
-
-            fs::write(local_include, new_include.as_ref()).unwrap();
-        }
+        let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+        let bindings_output = out_dir.join("bindings.rs");
+
+        let vita_headers_include = vitasdk.join("arm-vita-eabi").join("include");
+        let db = vitasdk.join("share").join("vita-headers").join("db");
+        assert!(
+            vita_headers_include.exists(),
+            "VITASDK's `include` directory does not exist"
+        );
+        assert!(db.exists(), "VITASDK's `db` directory does not exist");
+
+        vitasdk_sys_build_util::bindgen::generate(
+            &vita_headers_include,
+            &db,
+            &bindings_output,
+            &vitasdk_sys_manifest(),
+            true,
```

Comment:
Yeah I agree, I have no idea what to name it though, the best I thought was:

```rust
pub enum RunType {
    Standalone,
    BuildRs
}
```

As an alternative, [I bound the value to a variable before using it](https://github.com/vita-rust/vitasdk-sys/pull/21/files#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R60), improving readability. As an "internal" only API it may be enough, what do you think? If you'd rather go with the `enum` route, what do you think of the name?